shared/xccdf/services/ssh.xml

```diff
@@ -726,6 +782,61 @@ functions. The only SSHv2 hash algorithms meeting this requirement is SHA2.
 ossrg="SRG-OS-000250-GPOS-00093" stigid="040400" cui="3.1.13,3.13.11,3.13.8" />
 </Rule>
 
+<!-- similar to sshd_use_approved_ciphers, but without FIPS certification -->
+<Rule id="sshd_use_strong_ciphers" severity="medium" prodtype="rhel7,debian8">
+<title>Use Only Strong Ciphers</title>
+<description>Limit the ciphers to strong algorithms.
+Counter (CTR) mode is also preferred over cipher-block chaining (CBC) mode.
+The following line in <tt>/etc/ssh/sshd_config</tt>
+demonstrates use of those ciphers:
+<pre>Ciphers aes128-ctr,aes192-ctr,aes256-ctr</pre>
+<pre>chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr</pre>
+The man page <tt>sshd_config(5)</tt> contains a list of supported ciphers.
+</description>
+<ocil clause="ciphers are not configured or not using strong ciphers">
+Only approved ciphers should be used.  To verify that only approved
+ciphers are in use, run the following command:
+<pre>$ sudo grep Ciphers /etc/ssh/sshd_config</pre>
+The output should contain only those ciphers which are considered strong, namely,
+chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr
+</ocil>
+<rationale>
+Based on research conducted at various institutions, it was determined that the symmetric
+portion of the SSH Transport Protocol (as described in RFC 4253) has security weaknesses
+that allowed recovery of up to 32 bits of plaintext from a block of ciphertext that was
+encrypted with the Cipher Block Chaining (CBD) method. From that research, new Counter
+mode algorithms (as described in RFC4344) were designed that are not vulnerable to these
+types of attacks and these algorithms are now recommended for standard use.
+</rationale>
+<oval id="sshd_use_strong_ciphers" />
+<ref prodtype="debian" cis="9.3.11" />
+</Rule>
+
+<!-- similar to sshd_use_approved_macs, but without FIPS certification -->
+<Rule id="sshd_use_strong_macs" severity="medium" prodtype="rhel7,debian8">
+<title>Use Only Strong MACs</title>
+<description>Limit the MACs to strong hash algorithms.
+The following line in <tt>/etc/ssh/sshd_config</tt> demonstrates use
+of those MACs:
+<pre>MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,hmac-ripemd160</pre>
+</description>
+<ocil clause="MACs option is commented out or not using strong hash algorithms">
+Only approved MACs should be used.  To verify that only approved
+MACs are in use, run the following command:
+<pre>$ sudo grep -i macs /etc/ssh/sshd_config</pre>
+The output should contain only those MACs which are approved, namely,
```

Comment:
I would suggest switching the word "approved" to  " strong" in the few lines above.
Otherwise, who needs to approve it? The policy writing organization?